bigbluebutton-html5/imports/ui/components/toast/service.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+
+import Toast from './component';
+
+let lastToastId = null;
+const notify = (message, type = 'default', icon, options) => {
+  const settings = {
+    type,
+    ...options,
+  };
+
+  if (!toast.isActive(lastToastId)) {
```

Comment:
I've ran into issues with this thing. 
If you try to send a notification while another is active, it will now show up.
probably a debounce in favor of this.